README.md

```diff
@@ -1 +1,65 @@
-# Xpenda-App-v2
\ No newline at end of file
+[![](https://img.shields.io/badge/Microverse-Dozie%20Udeagha-blueviolet)](https://github.com/udeaghad)
+
+# Xpenda-App
+
+> The Ruby on Rails project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what. 
+
+I created an application that allows the user to:
+- register and log in, so that the data is private to them.
+- introduce new transactions associated with a category.
+- see the money spent on each category.
+
+
+## Built With
+
+- Ruby
+- Rails
+- Postgresql
+- CSS
+- Tailwind
+
+## Live Demo (if available)
+Click **[here](https://mighty-brushlands-87793.herokuapp.com/)** for live demo
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+### Setup
+ **clone this [repo](https://github.com/udeaghad/Xpenda-App-v2)**
+
+### Usage and Installation
+- Navigate to the directory `cd Xpenda-app` (case sensitive, make sure you type the correct path)
+- Run `bundle install` in your terminal (in case you need to update gems run `bundle update`)
+- To setup database and tailwind run the following commands:
+  - `rails db:create`
+  - `rails db:migrate`
+  - `rails db:seed` (to populate initial data from db/seeds.rb file)  
+- Run `rails s` on your terminal
+- Enter **http://localhost:3000/ or localhost:3000** on your browser to view the project.
+- **Congratulations! App should run successfully.**
+## Authors
+
+👤 **Dozie Udeagha**
+
+- GitHub: [@udeaghad](https://github.com/udeaghad)
+- Twitter: [@theodoz](https://twitter.com/theodoz)
+- LinkedIn: [@udeaghad](https://www.linkedin.com/in/dozie-udeagha/)
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/udeaghad/Xpenda-App-v2/issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Kindly give appropriate credit to [Gregoire Vella](https://www.behance.net/gregoirevella) on [Behance the author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) in the "Acknowledgements" section of the README file. :+1: